tools/src/linear-create-audit-issues/Configuration.ts

```diff
@@ -0,0 +1,199 @@
+          description: `Look at the JS/TS API and ensure it's consistent and properly typed. Pay extra attention to our public API rules (like adding \`Async\` suffix to the method names)`,
+          labelNames: ['Module', 'API'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - audit docs`,
+          description: `Ensure the docs for this module are up to date, comprehensive and easy to understand. Additionally ensure native english speaker has reviewed any changes.`,
+          labelNames: ['Module', 'Docs'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Web`,
+          description: `Ensure this module is properly implemented on the Web platform.`,
+          labelNames: ['Module', 'Web'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Android: rewrite to Kotlin`,
+          description: `Convert this module Android code to Kotlin.`,
+          labelNames: ['Module', 'Android'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Android: rewrite to SweetAPI`,
+          description: `Convert this module Android code to be SweetAPI-based`,
+          labelNames: ['Module', 'Android'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - iOS: rewrite to Swift`,
+          description: `Convert this module iOS code to Swift.`,
+          labelNames: ['Module', 'iOS'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - iOS: rewrite to SweetAPI`,
+          description: `Convert this module iOS code to be SweetAPI-based`,
+          labelNames: ['Module', 'iOS'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - JS tests`,
+          description: `Audit JS tests for this module. Check \`test-suite\` app and provide solid test coverage.`,
+          labelNames: ['Module', 'Testing'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - Android tests`,
+          description: `Convert this module iOS code to be SweetAPI-based`,
+          labelNames: ['Module', 'Android', 'Testing'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - iOS tests`,
+          description: `Convert this module iOS code to be SweetAPI-based`,
+          labelNames: ['Module', 'iOS', 'Testing'],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - examples screen`,
+          description: `Audit example screen by checking all module's functionalities are presented.`,
+          labelNames: ['Module', 'QA', `Testing`],
+        },
+        {
+          title: (moduleName: string) => `Audit ${moduleName} - compare with similar libraries`,
+          description: `Compare this module with other libraries available in the RN ecosystem. Check the differences and possibly narrow the functionalities gap.`,
```

Comment:
Also should check if there are any new possibilities offered by the new versions of iOS/Android.